api/client/swarm/join_token.go

```diff
@@ -12,11 +12,6 @@ import (
 	"golang.org/x/net/context"
 )
 
-const (
-	flagRotate = "rotate"
-	flagQuiet  = "quiet"
-)
```

Comment:
None of this matters very much. The reason I prefer not to use literal strings is that eventually someone adds some code that has to look up the flag by name (for example, calling `Changed`). If there's no constant to use, they duplicate the string, which makes it easy to make a mistake such as a typo in that string, and is harder to maintain than having a single definition of each flag's name.

Just do whatever you think is best. I don't mind having them in `opts.go`.